.github/workflows/macos-ci.yml

```diff
@@ -0,0 +1,68 @@
+name: 'MacOS - CI'
+
+on:
+  # Activate manually
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build
+    runs-on: macos-latest
```

Comment:
Might be better to specify `macos-10.15` specifically, but we can do that later. https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on